static/app/views/performance/traceDetails/traceViewDetailPanel.tsx

```diff
@@ -211,6 +212,34 @@ function EventDetails({detail, organization, location}: EventDetailProps) {
     .filter(isNotPerformanceScoreMeasurement)
     .sort();
 
+  const renderMeasurements = () =>  {
+      if(!detail.event)
+        {return null;}
+
+      const {measurements} = detail.event;
+
+      const measurementKeys = Object.keys(measurements ?? {})
+        .filter(name => Boolean(WEB_VITAL_DETAILS[`measurements.${name}`]))
+        .sort();
+
+      if (measurementKeys.length <= 0) {
+        return null;
+      }
+
+      return (
+        <Fragment>
+          {measurementKeys.map(measurement => (
+            <Row
+              key={measurement}
+              title={WEB_VITAL_DETAILS[`measurements.${measurement}`]?.name}
+            >
+              {`${Number(measurements?.[measurement].value.toFixed(3)).toLocaleString()}ms`}
```

Comment:
If the key access results in undefined, this would result in `NaN` being displayed in the UI
Also with the `ms` suffix, should we just use the `PerformanceDuration` component here to handle unit conversions when the measurement is a really long duration?